index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Mirwais Farahi</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <nav class="navbar">
+                <div class="logo">
+                    <a href="">My Logo</a>
+                </div>
+
+                <div class="menu">
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                    <div class="menu-line"></div>
+                </div>
+            </nav>
+        </header>
+        <main>
+            <section>
+                <h1>I’m Mirwais <br />
+                    Glad to see <br />you!</h1>
+                <p>
+                    I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <div class="contact">
+                    <h3>LET'S CONNECT</h3>
+                    <a class="twitter" href="#"><img src="/images/twitter.png" alt="twitter link"></a>
+                    <a class="linkedin" href="#"><img src="/images/Linkedin_icon.png" alt="linked link"></a>
+                    <a class="m" href="#"><img src="/images/M.png" alt="M"></a>
+                    <a class="github" href="#"><img src="/images/github.png" alt="github link"></a>
+                    <a class="doits" href="#"><img src="/images/doits.png" alt="doit"></a>
+                </div>
```

Comment:
- [ ] Let's change these multiple a tags within a div into one `ul` element containing multiple `li` elements, div should be used to separate unrelated content and a mixture of uls and lis would a friendly way to separate related ones.